driver-core/src/main/com/mongodb/internal/operation/ListIndexesOperation.java

```diff
@@ -127,8 +124,8 @@ public BatchCursor<T> execute(final ReadBinding binding) {
                     return createReadCommandAndExecute(retryState, binding, source, namespace.getDatabaseName(), getCommandCreator(),
                             createCommandDecoder(), transformer(), connection);
                 } catch (MongoCommandException e) {
-                    return rethrowIfNotNamespaceError(e, createEmptyBatchCursor(namespace, decoder,
-                            source.getServerDescription().getAddress(), batchSize));
+                    return rethrowIfNotNamespaceError(e,
+                            SingleBatchCursor.createEmptyBatchCursor(source.getServerDescription().getAddress(), batchSize));
```

Comment:
We generally use static imports for spots like this.

Reply:
Renamed the methods and done.